Makefile

```diff
@@ -240,7 +240,7 @@ coverage: .init
 	$(DOCKER_CMD) contrib/hack/coverage.sh --html "$(COVERAGE)" \
 	  $(addprefix ./,$(TEST_DIRS))
 
-test: .init build test-unit test-integration
+test: .init build test-unit # test-integration # turn off test-integration until integration tests are set up
```

Comment:
could we fix the CI system to use `make test-unit` instead of disabling what a human is going to want to use?

Reply:
My comments are a bit misleading, it appears. A human running `make test` will get the same failures when it comes time to run the integration tests. That defect is not limited to CI. I'll make it a priority to get integration tests working soon so that we can re-enable it for both humans and CI.

Reply:
In other words, I don't think it is worthwhile to spend time changing what the CI system is running.